inst/examples/e2e-test/R/themeProvider.R

```diff
@@ -0,0 +1,41 @@
+theme <- list(
+  palette = list(
+    themePrimary = "#8dd400",
+    themeLighterAlt = "#060800",
+    themeLighter = "#172200",
+    themeLight = "#2a3f00",
+    themeTertiary = "#557f00",
+    themeSecondary = "#7cba00",
+    themeDarkAlt = "#97d816",
+    themeDark = "#a6de35",
+    themeDarker = "#bce766",
+    neutralLighterAlt = "#323130",
+    neutralLighter = "#31302f",
+    neutralLight = "#2f2e2d",
+    neutralQuaternaryAlt = "#2c2b2a",
+    neutralQuaternary = "#2a2928",
+    neutralTertiaryAlt = "#282726",
+    neutralTertiary = "#c8c8c8",
+    neutralSecondary = "#d0d0d0",
+    neutralPrimaryAlt = "#dadada",
+    neutralPrimary = "#ffffff",
+    neutralDark = "#f4f4f4",
+    black = "#f8f8f8",
+    white = "#323130"
+  )
+)
+
+themeProviderUI <- function(id) {
+  ns <- NS(id)
+  div(
+    `data-cy` = ns("themeProviderContent"),
```

Comment:
I don't mind this solution but is there a reason to use `data-cy` instead of `id` in tests?

Reply:
Using `data-cy` is a suggested solution by Cypress itself so I've decided to take advantage of it